Gemfile

```diff
@@ -77,6 +77,7 @@ group :test do
   gem 'database_cleaner', '~> 1.6.1'
   gem 'email_spec', '~> 2.1.0'
   gem 'rspec-rails', '~> 3.6'
+  gem 'selenium-webdriver', '~> 3.10'
```

Comment:
This gem is on the custom Gemfile already, [here](https://github.com/AyuntamientoMadrid/consul/blob/master/Gemfile_custom#L4) —You can remove it from here or there, both are valid.